pubsub/aws/config.go

```diff
@@ -13,6 +13,8 @@ type (
 		aws.Config
 		QueueName           string `envconfig:"AWS_SQS_NAME"`
 		QueueOwnerAccountID string `envconfig:"AWS_SQS_OWNER_ACCOUNT_ID"`
+		// If SQS queue URL is provided it overrides QueueName and QueueOwnerAccountID
```

Comment:
Mind altering this to make the linter happy? The sentence just needs to start with the field name. 

For example: `QueueURL can be used instead of QueueName and QueueOwnerAccountID. If provided, the client will skip the "GetQueueUrl" call to AWS.`

Reply:
sure, will do.